### PR TITLE
Update virtual-appointments.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the licensing requirement

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10655

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/frontline/virtual-appointments.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The Virtual Appointments app provides a central hub for all your virtual appoint
...
+With Microsoft 365: A3, A5, E3, E5, F1, F3, Business Basic, Business Standard, Business Premium licenses, you can use basic Virtual Appointments capabilities that make it easy to schedule and join business-to-customer meetings. For example, you can schedule appointments in the Bookings calendar and external attendees can [join through a browser](browser-join.md) without having to download Teams. [Teams Premium](/microsoftteams/teams-add-on-licensing/licensing-enhance-teams) (Preview) unlocks advanced Virtual Appointments capabilities that your organization can use to manage and personalize the experience. These include a queue view of scheduled and on-demand appointments, SMS text notifications, custom waiting rooms, and analytics.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/virtual-appointments.md]:
```suggestion
With Microsoft 365 A3, A5, E3, E5, F1, F3, Business Basic, Business Standard, and Business Premium licenses, you can use basic Virtual Appointments capabilities that make it easy to schedule and join business-to-customer meetings. For example, you can schedule appointments in the Bookings calendar and external attendees can [join through a browser](browser-join.md) without having to download Teams. [Teams Premium](/microsoftteams/teams-add-on-licensing/licensing-enhance-teams) (Preview) unlocks advanced Virtual Appointments capabilities that your organization can use to manage and personalize the experience. These include a queue view of scheduled and on-demand appointments, SMS text notifications, custom waiting rooms, and analytics.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@LanaChin Please merge this PR. Thanks

**Comment by samanro** [comment]
Updating in separate PR: https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/18576

**Pull request closed without merging**